### PR TITLE
Complete the device adapter list for iEi Tank device.

### PR DESCRIPTION
**Pull request by bharani-zededa**
Signed-off-by: Bharani Chadalavada <bharani@zededa.com>

### CODE REVIEW

File: pkg/pillar/conf/AssignableAdapters/iEi.SER0.json [hunk 1]
```diff
@@ -1,5 +1,10 @@
 { "IoBundleList": [
     { "Type": 1, "Name": "eth0", "Members": [ "eth0" ], "Lookup": true },
     { "Type": 1, "Name": "eth1", "Members": [ "eth1" ], "Lookup": true },
+    { "Type": 255, "Name": "VGA", "Members": [ "VGA" ], "PciLong": "0000:00:02.0", "PciShort":"00:02.0" },
+    { "Type": 2, "Name": "USB", "Members": [ "USB0", "USB1", "USB2", "USB3", "USB4", "USB5", "USB6", "USB7" ], "PciLong": "0000:00:14.0", "PciShort":"00:14.0" },
+    { "Type": 255, "Name": "Thermal subsystem", "Members": [ "Thermal subsystem" ], "PciLong": "0000:00:14.2", "PciShort":"00:14.2" },
+    { "Type": 255, "Name": "Communication controller", "Members": [ "Communication controller" ], "PciLong": "0000:00:16.0", "PciShort":"00:16.0" },
+    { "Type": 255, "Name": "Serial controller", "Members": [ "Serial controller" ], "PciLong": "0000:00:16.3", "PciShort":"00:16.3" },
```
Comment (rvs) [inline pkg/pillar/conf/AssignableAdapters/iEi.SER0.json]: What's the point of enumerating all these 255 types? Which ones are we actively trying to manage via assignable adapters (e.g. VGA is clearly not re-assignable at this point without much more work in Dom0).

Comment (bharani-zededa) [reply pkg/pillar/conf/AssignableAdapters/iEi.SER0.json]: Is there a downside to enumerating all the adapters that we can assign (even if the immediate use case doesn't require)? As for your other question(s):

The one of immediate interest is "USB", but it is bundled with "Thermal subsystem". Per @nordmark I need to enumerate both so they both can be assigned together. 

Per @nordmark VGA assignment should now work.

I added "serial+communications" for completeness.

In any case, my understanding is that this list will move out of EVE to the controller near future. So, this is just an interim.

Comment (rvs) [reply pkg/pillar/conf/AssignableAdapters/iEi.SER0.json]: I am actually surprised that @nordmark feels VGA will work, but we can leave that aside. My general point is that we've never done this before with so many adapters that aren't either USB, serial or ethernet. In theory it should be safe to do this, but I don't know. However, as long as you've tested this on tank and it all seems to work -- I have no issue with this PR and we can merge it.

After all, the worst case scenario is that tank will get broken.

**Comment by eriknordmark** [comment]
@rvs @bharani-zededa I have never claimed that VGA assignment would work. It hasn't been tested.
From the perspective of the lspci output, the VGA adapter is a distinct controller. Hence assignment might work.

**Comment by bharani-zededa** [comment]
@nordmark, sure. I do remember you mention that it needs to be tested (I do plan to test it when I get a chance). I was only responding to @rvs comment that it won't work "without much work on Dom0". My understanding from our discussions was that we don't anticipate much wok on Dom0 (modulo any bugs or small issues). If that is not the case, then I guess I misunderstood.

**Pull request merged**